src/components/NavMenuMobileButtons/NavMenuMobileButtons.js

```diff
@@ -25,34 +26,39 @@ const styles = {
     backgroundColor: '#2B2B2B',
     padding: 0,
     margin: 0,
-    width: '50%',
+    width: '49%',
     textAlign: 'center',
     textDecoration: 'none',
     lineHeight: 'normal',
   },
   label: {
     fontSize: '16px',
-    margin: '0 0 0 10px',
-    textTransform: 'uppercase',
+    margin: '0 3px 0 5px',
     display: 'inline-block',
   },
   wrapper: {
-    width: '100%',
-    display: 'block',
+    alignItems: 'center',
+    display: 'flex',
+    justifyContent: 'center',
     margin: '0',
     padding: '1.75em 0',
+    width: '100%',
   },
   subscribeLinkWrapper: {
-    borderLeft: '1.25px solid #252525',
+    borderLeft: '1px solid #363636',
   },
   subscribeLinkLabel: {
-    width: '85px',
+    // width: '85px',
   },
   libraryCardLinkWrapper: {
-    borderRight: '1.25px solid #252525',
+    borderRight: '1px solid #363636',
+  },
+  shopLinkWrapper: {
+    display: 'flex',
+    justifyContent: 'center',
   },
   libraryCardLinkLabel: {
```

Comment:
You can remove this, too.

Reply:
In the places where `libraryCardLinkLabel` and `subscribeLinkLabel` are being used in the `_extend`, replace those variables with empty objects.

Reply:
thanks for your guidance on this one.